.github/workflows/nia-e2e.yaml

```diff
@@ -0,0 +1,49 @@
+name: Now in Android E2E
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, reopened, synchronize, labeled]
+    branches:
+      - main
+    paths:
+      - '.github/workflows/nia-*'
+      - 'nowinandroid/**'
+
+permissions:
+  contents: read
+  
+jobs:
+  nia-e2e:
+    name: Setup Anbox and Run E2E Tests
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'run-nia-e2e-tests')
+    timeout-minutes: 60
+    defaults:
+      run:
+        working-directory: ./nowinandroid
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+    
+    - name: Setup Android Dependencies
+      uses: ./.github/actions/setup-android
+    
+    - name: Setup Anbox Cloud Appliance
+      uses: ./.github/actions/setup-anbox
```

Comment:
As discussed we should keep this as bare and as close to the official steps we document for the action and in our documentation as possible. These are nice additions but not necessarily something which will fit for people looking at the example. Even if that makes the workflow longer that is what we're looking for to show as an example. If there things we can do in the official action to reduce boilerplate, lets do that instead.